### PR TITLE
Fixed CA1304, CA1308, CA2213 and did some refactorings

### PR DESCRIPTION
**Pull request by mertyeter**
Fixed 
**CA1304:** Specify CultureInfo, 
**CA1308:** Normalize strings to uppercase, 
**CA2213:** Disposable fields should be disposed

### CODE REVIEW

File: src/SQLQueryStress/Form1.cs [hunk 1]
```diff
@@ -238,8 +238,8 @@ private void backgroundWorker1_RunWorkerCompleted(object sender, RunWorkerComple
 
         private void AutoSaveResults(string resultsAutoSaveFileName)
         {
-            string extension = Path.GetExtension(resultsAutoSaveFileName).ToLower();
-            if (extension == ".csv")
+            string extension = Path.GetExtension(resultsAutoSaveFileName).ToUpperInvariant();
```
Comment (TheConstructor) [inline src/SQLQueryStress/Form1.cs:241]: Is `.ToUpperInvariant()` needed, when the comparison now is case insensitive? Also it previously was `.ToLower()`

File: src/SQLQueryStress/Form1.Designer.cs [hunk 2]
```diff
@@ -30,9 +30,11 @@ protected override void Dispose(bool disposing)
             }
             else
             {
-                if (disposing && (components != null))
+                if (disposing)
```
Comment (ErikEJ) [inline src/SQLQueryStress/Form1.Designer.cs:33]: Revert changes to this designer file

File: src/SQLQueryStress/LoadEngine.cs [hunk 3]
```diff
@@ -376,137 +376,135 @@ public void StartLoadThread(Object token)
...
                     });
                     //do the work
-                    using (var conn = _queryComm.Connection)
+                    using var conn = _queryComm.Connection;
+                    SqlInfoMessageEventHandler handler = GetInfoMessages;
+
+                    for (var i = 0; i < _iterations && !runCancelled; i++)
                     {
-                        SqlInfoMessageEventHandler handler = GetInfoMessages;
+                        Exception outException = null;
```
Comment (ErikEJ) [inline src/SQLQueryStress/LoadEngine.cs:384]: This is very hard to read, please limit to FxCop changes for now

File: src/SQLQueryStress/Form1.cs (hunk 1 above)
Comment (ErikEJ) [reply src/SQLQueryStress/Form1.cs:241]: This is the FxCop recommended approach

File: src/SQLQueryStress/Form1.Designer.cs [hunk 4]
```diff
@@ -30,11 +30,9 @@ protected override void Dispose(bool disposing)
             }
             else
             {
-                if (disposing)
+                if (disposing && (components != null))
```
Comment (ErikEJ) [inline src/SQLQueryStress/Form1.Designer.cs]: Undo designer changes

**Pull request merged**